src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDate by;
+
+    /**
+     * Parameterized constructor to create a Deadline
+     * @param description description of the task to be created
```

Comment:
can consider having an empty line between description and parameter section for better readability in the javadoc comments :-)